packages/samples/headless-react/src/components/url-manager/url-manager.ts

```diff
@@ -1,11 +1,11 @@
-import {buildUrlManager, Engine} from '@coveo/headless';
+import {buildUrlManager, SearchEngine} from '@coveo/headless';
 
 /**
  * Search parameters, defined in the url's hash, should not be restored until all components are registered.
  *
  * Additionally, a search should not be executed until search parameters are restored.
  */
-export function bindUrlManager(engine: Engine) {
+export function bindUrlManager(engine: SearchEngine) {
```

Comment:
Should we add `@param NAME - description` to functions like these?

```suggestion
 * Additionally, a search should not be executed until search parameters are restored.
 * @param engine - The search engine.
 */
export function bindUrlManager(engine: SearchEngine) {
```